src/Frontend/util/package-search-api.ts

```diff
@@ -63,25 +63,30 @@ export class PackageSearchApi {
   }
 
   public async getPackages(
-    packageName: string,
+    packageName?: string,
```

Comment:
let's change this to have the same signature as other methods in this class:
```suggestion
     props: PackageInfo
```
and then use `this.sanitize` to sanitize your input.

Reply:
Done.